apps/web-tools/trustchain/components/App.tsx

```diff
@@ -83,9 +84,20 @@ const App = () => {
         ? "PROD"
         : "MIXED";
 
+  const [deviceInteractionVisible, setDeviceInteractionVisible] = useState(false);
+  const callbacks = useMemo(
+    () => ({
+      onStartRequestUserInteraction: () => setDeviceInteractionVisible(true),
+      onEndRequestUserInteraction: () => setDeviceInteractionVisible(false),
```

Comment:
It will not work for all case, in this case it's works because it's very atomical and driven by click but on the LLD UI sometimes  you don't have the same amount of interactions due to the current flow you are. It means that sometimes callbacks are not called

Reply:
I'm not sure I understand, let's discuss today.